.travis.yml

```diff
@@ -1,18 +1,24 @@
 if: (branch = master) OR (type = pull_request) OR (tag IS present)
+language: rust
+rust: 1.39.0
+cache: cargo
+before_cache:
+  - find ./target/debug -type f -maxdepth 1 -delete
+  - rm -fr ./target/debug/{deps,.fingerprint}/{*ra_*,*test*,*tools*,*gen_lsp*}
+  - rm -f  ./target/.rustc_info.json
+env:
+  - CARGO_INCREMENTAL=0
 matrix:
   include:
     - stage: WarmUp
       name: "Build"
-      language: generic
       install:
         - sudo apt-get update && sudo apt-get install -y python-pip && sudo pip install awscli
-        - echo -e $GITLAB_PRIVATE_KEY > .ssh/id_rsa && chmod 0500 .ssh/id_rsa
```

Comment:
This is so much more straight forward with the docker image build in dex-solver. Thumbs up